src/Nethermind/Nethermind.JsonRpc/WebSockets/JsonRpcWebSocketsClient.cs

```diff
@@ -20,45 +20,76 @@
 using System.Text;
 using System.Threading.Tasks;
 using Nethermind.Core.Extensions;
+using Nethermind.JsonRpc.Modules;
+using Nethermind.JsonRpc.Modules.Subscribe;
 using Nethermind.Serialization.Json;
 using Nethermind.WebSockets;
 
 namespace Nethermind.JsonRpc.WebSockets
 {
-    public class JsonRpcWebSocketsClient : IWebSocketsClient
+    public interface IJsonRpcDuplexClient : IDisposable
```

Comment:
move to separate file and to Nethermind.JsonRpc namespace, it is not WebSocket specific (can be also interface for IPC)